build.cake

```diff
@@ -380,4 +380,19 @@ Task("nuget-validation")
 	}
 
 });
+
+Task ("docs-api-diff")
```

Comment:
What starts this task? Will it always run? Or is it manual?

Reply:
it'll need to be added into ci the same as the nuget-validation task